server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -252,18 +264,28 @@ mkCompExpInp colTy =
         )
       ]
 
-    -- Geometry related ops
-    stDWithinOpInpVal =
-      InpValInfo (Just stDWithinDesc) "_st_d_within" Nothing $ G.toGT stDWithinInpTy
-    stDWithinDesc =
-      "is the column within a distance from a geometry value"
+    stDWithinGeoOpInpVal desc ty =
+      InpValInfo (Just desc) "_st_d_within" Nothing $ G.toGT ty
```

Comment:
Use `stDWithinGeoDesc` here instead of `desc` argument.

Reply:
Done!